test/e2e/resource_test.go

```diff
@@ -183,6 +185,75 @@ var _ = ginkgo.Describe("[resource-status collection] resource status collection
 		})
 	})
 
+	ginkgo.Context("ServiceStatus collection testing", func() {
+		var serviceNamespace, serviceName string
+		var service *corev1.Service
+
+		ginkgo.BeforeEach(func() {
+			policyNamespace = testNamespace
+			policyName = deploymentNamePrefix + rand.String(RandomStrLength)
+			serviceNamespace = testNamespace
+			serviceName = policyName
+
+			service = helper.NewService(serviceNamespace, serviceName)
+			service.Spec.Type = corev1.ServiceTypeNodePort
+			policy = helper.NewPropagationPolicy(policyNamespace, policyName, []policyv1alpha1.ResourceSelector{
+				{
+					APIVersion: service.APIVersion,
+					Kind:       service.Kind,
+					Name:       service.Name,
+				},
+			}, policyv1alpha1.Placement{
+				ClusterAffinity: &policyv1alpha1.ClusterAffinity{
+					ClusterNames: framework.ClusterNames(),
+				},
+			})
+		})
+
+		ginkgo.BeforeEach(func() {
+			framework.CreateService(kubeClient, service)
+			ginkgo.DeferCleanup(func() {
+				framework.RemoveService(kubeClient, serviceNamespace, serviceName)
+			})
+		})
+
+		ginkgo.It("NodePort service apply status collection testing", func() {
+			nodePorts := sets.NewInt32()
+			// simulate the update of the service status in member clusters.
+			ginkgo.By("Update service status in member clusters", func() {
```

Comment:
Those description seems not fit with the actual facts.

Reply:
ok